strato-run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+NODE_HOST="localhost" \
+  OAUTH_DISCOVERY_URL="OAUTH_DISCOVER_URL_PLACEHOLDER" \
+  OAUTH_CLIENT_ID="OAUTH_CLIENT_ID_PLACEHOLDER" \
+  OAUTH_CLIENT_SECRET="OAUTH_CLIENT_SECRET_PLACEHOLDER" \
+  SENDGRID_API_KEY="SENDGRID_API_KEY_PLACEHOLDER" \
+  VAULT_URL="https://vault.blockapps.net:8093" \
+  network='mercata-hydrogen' \
+  STRIPE_PAYMENT_SERVER_URL="https://payments.mercata-testnet2.blockapps.net" \
```

Comment:
I would prefer not sharing the urls that explicitly to the public, especially since we are planning to remove these from the run script.
What do you think?

Reply:
That makes sense. It looks like it'll be removed anyway, so I'll remove that line